src/PlotlyEditor.js

```diff
@@ -69,7 +69,7 @@ class PlotlyEditor extends Component {
           this.props.afterUpdateTraces(payload);
         }
         if (this.props.onUpdate) {
-          this.props.onUpdate();
+          this.props.onUpdate(graphDiv.data, graphDiv.layout);
```

Comment:
in this iteration, we mutate and pass along. In the next iteration, we'll use `immutability-helper` to NOT mutate and pass along copies :)